data-non-standard.js

```diff
@@ -1438,6 +1472,34 @@ exports.tests = [
   separator: 'after'
 },
 {
+  name: 'Object.observe',
+  link: 'https://arv.github.io/ecmascript-object-observe/',
+  category: 'draft',
```

Comment:
Also, remove this line
![default](https://cloud.githubusercontent.com/assets/2213682/11565622/f81f88ee-9a08-11e5-9ca9-b1c7d214195e.png)